include/rdma/fi_log.h

```diff
@@ -35,6 +35,8 @@
 #ifndef _FI_LOG_H_
 #define _FI_LOG_H_
 
+#include "config.h"
```

Comment:
My only care is that this single place differs from probably a hundred other places where config.h is included.  If the solution is to remove the #if in the other places, then I'm fine with this commit.